packages/spright-components/src/chat/message/index.ts

```diff
@@ -10,6 +17,12 @@ declare global {
     }
 }
 
+/**
+ * SprightChatMessage configuration options
+ * @public
+ */
+export type ChatMessageOptions = FoundationElementDefinition & StartEndOptions;
```

Comment:
I'm not familiar with the our/FAST's StartEndOptions pattern. It looks like this follows what the tab does by exporting this type and using it from the template. But I was expecting to see the pattern in more other components. @rajsite could you double check that it's being applied correctly?

Reply:
I learned more about this pattern while I contributed to the PR and it looks good to me. Resolving.